primefaces/src/main/resources/META-INF/resources/primefaces/datepicker/0-datepicker.js

```diff
@@ -1363,7 +1368,7 @@
                     yearOptions.push(i);
                 }
 
-                return '<select class="ui-datepicker-year">' + this.renderTitleOptions('year', yearOptions, year) + '</select>';
+                return '<select class="ui-datepicker-year" aria-label="'+this.options.locale.select+' '+this.options.locale.year+'">' + this.renderTitleOptions('year', yearOptions, year) + '</select>';
```

Comment:
Why not simply do this?
```suggestion
                return '<select class="ui-datepicker-year" aria-label="'+this.options.locale.year+'">' + this.renderTitleOptions('year', yearOptions, year) + '</select>';
```